fe/fe-core/src/test/java/com/starrocks/sql/plan/CTEPlanTest.java

```diff
@@ -52,15 +52,13 @@ public void testFromUseCte() throws Exception {
                 "select * from (with x1 as (select * from t1) select * from x1 join x0 on x1.v4 = x0.v1) tt";
         String plan = getFragmentPlan(sql);
         System.out.println(plan);
-        Assert.assertTrue(plan.contains(" 8:HASH JOIN\n" +
-                "  |  join op: INNER JOIN (PARTITIONED)\n" +
+        Assert.assertTrue(plan.contains("  7:HASH JOIN\n" +
```

Comment:
Why it affects the execution plan

Reply:
I will remove this, and rebase on https://github.com/StarRocks/starrocks/pull/2596.